ra/ra.go

```diff
@@ -897,16 +897,35 @@ func (ra *RegistrationAuthorityImpl) recheckCAA(ctx context.Context, authzs []*c
 			ch <- err
 		}(authz)
 	}
-	var caaFailures []string
-	for _ = range authzs {
-		if err := <-ch; berrors.Is(err, berrors.CAA) {
-			caaFailures = append(caaFailures, err.Error())
-		} else if err != nil {
-			return err
+	var subErrors []berrors.SubBoulderError
+	var firstBadIdent *identifier.ACMEIdentifier
+	for _, authz := range authzs {
+		err := <-ch
+		if err != nil {
+			if firstBadIdent == nil {
```

Comment:
I think we can probably do without `firstBadIdent` here, below where we want to access the first ident I think we could just use `subErrors[0].Identifier`?

Reply:
Oh yeah, that's much easier :man_facepalming: Fixed in 7029f60